src/diart/models.py

```diff
@@ -6,20 +6,46 @@
 
 try:
     import pyannote.audio.pipelines.utils as pyannote_loader
+    from pyannote.audio.utils.powerset import Powerset
 
     _has_pyannote = True
 except ImportError:
     _has_pyannote = False
 
 
+class PowersetAdapter(nn.Module):
+    def __init__(self, segmentation_model: nn.Module):
+        super().__init__()
+        self.model = segmentation_model
+        specs = self.model.specifications
+        max_speakers_per_frame = specs.powerset_max_classes
+        max_speakers_per_chunk = len(specs.classes)
+        self.powerset = Powerset(max_speakers_per_chunk, max_speakers_per_frame)
+
+    @property
+    def sample_rate(self) -> int:
+        return self.model.hparams.sample_rate
+
+    @property
+    def duration(self) -> float:
+        return self.model.specifications.duration
```

Comment:
@hbredin I got it to work with the following workaround:

```suggestion
    @property
    def specifications(self):
        return self.model.specifications

    @property
    def audio(self):
        return self.model.audio
```

This is very inconvenient for future wrappers that we may want to implement, but let's leave it like this for now. I'll take care of removing all these default values when I refactor the config. For now it's not a big deal cause it only impacts pretty advanced custom usage.